pkg/driver/controller_test.go

```diff
@@ -1220,6 +1383,66 @@ func TestControllerExpandVolume(t *testing.T) {
 	}
 }
 
+func TestIsShareableVolume(t *testing.T) {
+	testCases := []struct {
+		name       string
+		shareable  bool
```

Comment:
it will be good if you can rename this to `expected`

Reply:
Fixed.